src/routes/auth/login.ts

```diff
@@ -131,6 +132,13 @@ async function loginAccount({ body, headers }: Request, res: Response): Promise<
   }
 
   if (mfa_enabled) {
+    await request(updateTicketExpiration, {
+      email,
+      ticket_expires_at: new Date(
+        +Date.now() + 60 * 60 * 1000
+      )
+    })
```

Comment:
Is it better if we insert a new ticket instead of re-using an old ticket by just updating its' `ticket_expires_at` value?